### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by oprypin**

### CODE REVIEW

File: get_github_repos.py [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-exclude = {line.split(" ", 1)[0] for line in Path("exlude_repos.txt").read_text().splitlines()}
+exclude = {line.split(" ", 1)[0] for line in Path("exclude_repos.txt").read_text().splitlines()}
 
 
 def get_repos():
     from github import Github
 
     github = Github(TOKEN)
-    search = github.search_code("mkdocstrings+filename:mkdocs.yml")
+    search = github.search_code("mkdocstrings filename:mkdocs.yml")
```
Comment (pawamoy) [inline get_github_repos.py:16]: Ah, I was wondering why it didn't work anymore. After I created the mkdocstrings org on GitHub, the script was returning the org only, nothing else.

**Comment by pawamoy** [comment]
So, I guess you removed/commented all the repos that failed at "building current" with the latest 0.14.0b1 version of mkdocstrings (and kept only 30 of working repos)?

**Comment by oprypin** [comment]
Yes.
Being conscious about GitHub's server farms 😳

Actually ended up being 32 repos. I did the cutoff right before a big chunk of failing repos.

**Comment by pawamoy** [comment]
OK, thanks!

**Pull request merged**